apps/admin/backend/schema.sql

```diff
@@ -42,13 +42,23 @@ create table cvrs (
   ballot_type text not null 
     check (ballot_type = 'absentee' or ballot_type = 'precinct' or ballot_type = 'provisional'),
   batch_id text not null,
-  batch_label text not null,
   party_id text,
   precinct_id text not null,
   scanner_id text not null,
   sheet_number integer check (sheet_number is null or sheet_number > 0),
   votes text not null,
   created_at timestamp not null default current_timestamp,
+  foreign key (election_id) references elections(id)
+    on delete cascade,
+  foreign key (batch_id, scanner_id) references scanner_batches(batch_id, scanner_id)
+);
+
+create table scanner_batches (
```

Comment:
I can definitely see an argument for tossing away this commit and just leaving the batch data not normalized, but i wanted to make it cleaner and it's possible we will want additional batch metadata and don't want to shove it into the CVR table.